spec/models/parser_version_spec.rb

```diff
@@ -44,4 +44,15 @@
       parser_version.harvest_jobs('finished')
     end
   end
+
+  describe '#source' do
+    RSpec.configure { |c| c.include ActiveResourceMockHelper }
```

Comment:
I kind of think that this should be included to the rspec configure block rather than within the tests.

Reply:
done